/*
 * This file was automatically generated by EvoSuite
 * Wed Apr 20 11:29:55 GMT 2022
 */

package org.apache.activemq.artemis.api.core;

import org.junit.Test;
import static org.junit.Assert.*;
import org.apache.activemq.artemis.api.core.ActiveMQInternalErrorException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.evosuite.runtime.mock.java.lang.MockException;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class ActiveMQInternalErrorException_ESTest extends ActiveMQInternalErrorException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      ActiveMQInternalErrorException activeMQInternalErrorException0 = new ActiveMQInternalErrorException();
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      MockException mockException0 = new MockException("");
      ActiveMQInternalErrorException activeMQInternalErrorException0 = new ActiveMQInternalErrorException("", (Throwable) mockException0);
      ActiveMQInternalErrorException activeMQInternalErrorException1 = new ActiveMQInternalErrorException((String) null, (Exception) activeMQInternalErrorException0);
      assertFalse(activeMQInternalErrorException1.equals((Object)activeMQInternalErrorException0));
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      ActiveMQInternalErrorException activeMQInternalErrorException0 = new ActiveMQInternalErrorException((String) null);
  }
}
